### PR TITLE
adding example t_ex6_1 to rtd example file

### PR DESCRIPTION
**Pull request by mjburton11**
Ready for review. I'm trying to improve the documentation on the readthedocs page.  Here's a start!

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -3,4 +3,12 @@ Examples
 
 
 Example 1
-=========
\ No newline at end of file
+=========
+This example comes from Section 6 of the `GP Fitting <http://web.mit.edu/~whoburg/www/papers/gp_fitting.pdf>`_, by W. Hoburg et. at.
```
Comment (whoburg) [inline docs/source/examples.rst]: 1. "of the `GP Fitting" should be "of `GP Fitting", or "of the paper `Fitting geometric programming models to data"
2. et. at --> et. al.

**Comment by whoburg** [comment]
Thanks @mjburton11, great start. Made a few small suggestions/corrections inline. Push another commit to this branch to fix those?

**Comment by mjburton11** [comment]
Ready for review.  Just fixed the typos @whoburg commented on.

**Comment by whoburg** [comment]
http://gpfit.readthedocs.org/en/latest/examples.html

**Pull request merged**